book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] [REQUIRED] Great job so far @shasherazi , however, don't forget that a `book` has many rentals therefore a method for adding rentals should be present. Take a look at the example in the [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) here.
Kindly add a method to add rentals.